src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaFigure {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    abstract void toInformation();
```

Comment:
You don`t need this method cause you have toString()

Reply:
Part of task" 
To 'draw' means to print out all information about a figure using System.out.println() (you shouldn't override the toString() method for this).
Did I misunderstand this?